src/Providers/EntityManagerProvider.php

```diff
@@ -45,7 +45,7 @@ public function register()
                 $config->setAutoGenerateProxyClasses(Common\Proxy\AbstractProxyFactory::AUTOGENERATE_ALWAYS);
             }
 
-            $config->addFilter("family", "\Choredo\FamilyFilter");
+            $config->addFilter("family", "\Choredo\Entities\Filters\FamilyFilter");
```

Comment:
Does the class constant work here? 

`Choredo\Entities\Filters\FamilyFilter::class`

That way if you end up moving it the reference holds and you don't have to manually remember to change this.